.github/workflows/run-php-cs-fixer.yml

```diff
@@ -15,14 +15,14 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: shivammathur/setup-php@15c43e89cdef867065b0213be354c2841860869e
+    - uses: shivammathur/setup-php@v2
       with:
-        php-version: '8.1'
+        php-version: '8.2'
```

Comment:
I ask out of curiosity, will the MoonGuard update fix the PHP version to 8.2? I understand that this version of MoonGuard will only be compatible with Laravel 10.

Reply:
BTW, we should mention that on the docs.

Reply:
yep, you are right 😄